### PR TITLE
[SOLP-2812] add analyze mpr tool description

### PR DESCRIPTION
**Pull request by mikita-haurykau**

### CODE REVIEW

**Comment by tknippenberg** [comment]
Can we get this PR merged? 
I'm aware the documentation mentions it's "available from 10.17" (and that's what we want it to say in the public docs), but actually it has been available for a longer time already. 
As we need to send out some mailings in the coming days where we would like to refer to this documentation, I'd appreciate it if we can already publish this Docs change now.

**Comment by quinntracy** [comment]
Yes, we can merge it today.

________________________________

**Review by quinntracy — APPROVED** [review]
Approved doc placement

File: content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+---
+
+## Introduction
+
+The `mx analyze-mpr` command enables you to show information about the MPR file in the form of plaintext. 
+
+This command is available as of Mendix 10.17.
+
+{{% alert color="info" %}}
+This tool shipped with Mendix 10 can also be used to analyze MPR files created in Mendix 9.24 LTS.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md]: Which tool do we mean here? Command-line tool?

File: content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+{{% /alert %}}
+
+{{% alert color="warning" %}}
+The exact output of this tool may change. You cannot rely on this format staying consistent over time.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md]: I am not sure I understand what is meant in this warning. Could you please rewrite?

**Review by MariaShaposhnikova — CHANGES_REQUESTED** [review]
Please don't merge before comments are resolved.

File: content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md (hunk 1 above)
Comment (tknippenberg) [reply content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md]: This specific command in `mx.exe` is "available as of 10.17" but can be used on older MPR versions.

File: content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md (hunk 2 above)
Comment (tknippenberg) [reply content/en/docs/refguide/general/mx-command-line-tool/analyze-mpr.md]: The intention is to say that this is not a "public API" where customers can count on the format not changing. They should not build tooling on top of this output, as the output may change over time.
I hope with that input you can make a suggestion for the text.

**Review by MariaShaposhnikova — APPROVED** [review]

**Pull request merged**